utils/s2n_random.c

```diff
@@ -60,14 +56,19 @@
 
 static int entropy_fd = UNINITIALIZED_ENTROPY_FD;
 
-static __thread struct s2n_drbg per_thread_private_drbg = {0};
-static __thread struct s2n_drbg per_thread_public_drbg = {0};
-
-static void *zeroed_when_forked_page = NULL;
-static int zero = 0;
+struct s2n_rand_state {
+    uint64_t cached_fgn;
```

Comment:
This is just a random (pun intended) occurrence ... but why not spell out fgn everywhere it appears? Is the brevity really worth it? I don't think it's a common term that a new reader would instantly recognize, so I'd lean towards just spelling it out for clarity. It's a bit hypocritical because DRBG isn't spelled out, but that's an industry term, and maybe it should be spelled out!

Reply:
Just for brevity. Changed it to the full form. So, we can see how it looks.